app/views/notifications/_global_bar.html.erb

```diff
@@ -25,16 +26,24 @@
     <div class="global-bar-message-container">
     <p class="global-bar-message">
       <% if title %>
-        <span class="global-bar-title"><%= title %></span>
+        <% if title_href %>
+          <a class="global-bar-title govuk-link" href="<%= title_href %>"><%= title %></a>
+        <% else %>
+          <span class="global-bar-title"><%= title %></span>
+        <% end %>
       <% end %>
 
-      <% if link_href && link_text %>
-        <%= link_to(
-          link_text,
-          link_href,
-          rel: "external noreferrer",
-          class: "govuk-link js-call-to-action"
-        ) %>
+      <% if link_text %>
+        <% if link_href %>
+          <%= link_to(
+            link_text,
+            link_href,
+            rel: "external noreferrer",
+            class: "govuk-link js-call-to-action"
```

Comment:
Does this need to go on the new title link?

Reply:
Good spot - done 👍

Reply:
Thanks - do we also need https://github.com/alphagov/static/pull/1936/files#diff-e17896925164fa4471101aca24b30733R41?

Reply:
Hm, not sure that's accurate any more as the link isn't to an external site. Tempted to leave as is and remove in a separate PR so I can double check it's ok to remove with a performance analyst